CourseApp/FunctionCalculator.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class FunctionCalculator
+    {
+        public double Calculate(double b, double x)
+        {
+            double top = 1 + Math.Pow(Math.Sin(Math.Pow(b, 3) + Math.Pow(x, 3)), 2);
+            double bottom = Math.Pow(Math.Pow(b, 3) + Math.Pow(x, 3), 1 / 3);
+            double y = top / bottom;
+            return y;
+        }
+
+        public List<double> CalculateTaskA(double xStart, double xEnd, double dX, double b)
```

Comment:
Can we seperate specific tasks from `FunctionCalculator`?

According to `SRP`, `FunctionCalculator` should do only one think and should not know about specific tasks such as `CalculateTaskA` and `CalculateTaskB`.